lib/components/SelectionControl/ToggleSwitch/toggle-switch.scss

```diff
@@ -0,0 +1,89 @@
+@import "@quill/color.scss";
+@import "@quill/color.opacity.scss";
+@import "@quill/others.scss";
+@import "@quill/border.scss";
+@import "@quill/motion.scss";
+@import "@quill/size.scss";
+@import "@quill/spacing.scss";
+@import "@quill/color.semantic.scss";
+
+:root {
+    .theme--light {
+        --switch-background-color: var(--core-color-opacity-black-200);
+        --switch-active-color: var(--core-color-opacity-black-400);
+        --switch-enabled: var(--core-color-opacity-green-600);
+        --switch-active-enabled-color: var(--core-color-solid-green-900);
+        --switch-hover-enabled-color: var(--core-color-opacity-green-800);
+    }
+    .theme--dark {
+        --switch-background-color: var(--core-color-opacity-white-400);
+        --switch-active-color: var(--core-color-opacity-white-200);
+        --switch-enabled: var(--core-color-opacity-green-800);
+        --switch-active-enabled-color: var(--core-color-opacity-green-400);
+        --switch-hover-enabled-color: var(--core-color-opacity-green-600);
```

Comment:
is this a temporally solution for dark and light theme?

Reply:
Yes it is , will be replaced once we get semantic from design teams